Components/SignUp.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from "react";
+import { Text, View, TextInput } from "react-native";
+import { Button } from "native-base";
+import api from "../Utils/apiUtils";
+import styles from "./Styling/formStyling";
+import HeaderBar from "./HeaderBar";
+import { ScrollView } from "react-native-gesture-handler";
+
+class SignUp extends Component {
+  state = {
+    user_id: null,
+    firstName: "Enter your first name",
+    surname: "Enter your surname",
+    password: "Enter your password",
+    confirmPassword: "Confirm your password"
+  };
+
+  onPressButton = () => {
+    const { firstName, surname } = this.state;
+    api
+      .signUp(firstName, surname)
+      .then(({ user }) => {
+        this.setState({
+          user_id: user.id,
+          firstName: "Enter your first name here",
```

Comment:
```suggestion

```suggestion
          firstName: "Enter your first name here",
```